src/_includes/core-nodes/udp-in-use-case.md

```diff
@@ -0,0 +1,51 @@
+## What is udp-in node in Node-RED?
+
+The UDP-In node in Node-RED enables the reception of UDP messages from remote devices or services. It acts as a listener, waiting for incoming UDP packets on a specified port. This functionality is crucial for real-time applications, such as IoT data ingestion and network communication. Whether you're receiving sensor data from IoT devices or communicating with other networked systems, the UDP-In node seamlessly integrates UDP communication into your Node-RED workflows, providing a lightweight and efficient solution for data reception.
+
+### What is Udp?
+
+UDP (User Datagram Protocol) is a connectionless protocol in the Internet Protocol suite. It transmits data packets, or datagrams, without establishing a connection, prioritizing speed over reliability. Commonly used for real-time applications like video streaming, online gaming, and VoIP. UDP's simplicity reduces latency but doesn't guarantee delivery of data packets. For more information refer to [UDP MDN Docs](https://developer.mozilla.org/en-US/docs/Glossary/UDP)
+
+## Configuring UDP-In Node 
```

Comment:
Have you tried this on FlowFuse?

Reply:
No, I haven't tried it on the FlowFuse cloud; I've tried it on the locally running Node-RED instance.

Reply:
Please update the documentation with a line that explains why these aren't available on FFC